lib/cucumber/cli/argument_parser.js

```diff
@@ -21,9 +22,37 @@ function ArgumentParser(argv) {
                          shortenedOptionDefinitions,
                          args,
                          0);
+      options = self.processRemainingOptions(options);
```

Comment:
Instead of doing this inside of argsToOptions I think this parsing makes more sense in getSuppliedPaths. That way you won't need to reassign `options.argv.remain` but instead just parse it once and convert anything that starts with a `@` to the content of that file.